config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/AnnotableEntity.java

```diff
@@ -50,14 +50,14 @@
         if (annotations != null) {
             AnnotatedEntity<GatewayEntity> annotatedEntity = new AnnotatedEntity(this);
             annotations.forEach(annotation -> {
-                if (ANNOTATION_TYPE_BUNDLE.equalsIgnoreCase(annotation.getType())) {
-                    annotatedEntity.setTags(annotation.getTags());
+                if (BUNDLE.equalsIgnoreCase(annotation.getType())) {
                     annotatedEntity.setEntityType(EntityTypes.ENCAPSULATED_ASSERTION_TYPE);
```

Comment:
Though it is not part of this PR, why did we hard-code this? Rather, it should be something like: this.getType().

Reply:
This will be taken care in separate story for enabling service bundle